src/BlockLink.js

```diff
@@ -0,0 +1,15 @@
+import styled from 'styled-components'
+import Link from './Link'
+
+const BlockLink = styled(Link)`
+  display: block;
+  color: inherit;
+  text-decoration: inherit;
```

Comment:
This could probably be `text-decoration: none` and I don't think the `:hover` is required if this is built from an `<a>` tag instead of the Link component

Reply:
@jxnblk, make sense with `text-decoration: none;`. Updated per suggestion. ✌️